zerver/openapi/zulip.yaml

```diff
@@ -7035,7 +7035,9 @@ paths:
                     **Changes**: In Zulip 7.0 (feature level 177), narrows gained support
                     for three new filters related to direct messages: `is:dm`, `dm` and
                     `dm-including`; replacing and deprecating `is:private`, `pm-with` and
-                    `group-pm-with` respectively.
+                    `group-pm-with` respectively. Additionally, `is:unread` can now be
+                    applied more efficiently to improve message processing speed and
+                    optimize performance.
```

Comment:
Why is this added to the feature level 177 changelog entry?

Reply:
It was my mistake. I'll make it its own section

Reply:
This should come before the **Changes** note, and should probably be a more like a tip/suggestion than noting that the existence of the `is:unread` filter. Something like ...

> Note that when adding the `read` flag to messages, clients should consider including a narrow with the `is:unread` filter as an optimization that takes advantage of the fact that the server has a database index for unread messages.

Reply:
@JohnLu2004 - Why was this removed from the changes? The issue this is resolving specifically notes that this endpoint is where the documentation update/note needs to be added.

The addition to the construct a narrow page is less important.

Reply:
Hi Lauryn,

I misinterpreted what you meant. I've updated the link to work and put it in both the ``Construct a Narrow`` webpage and  ``Update personal message flags for narrows`` in my updated commit.

Here is what it looks like in ``Construct a Narrow``
![image](https://github.com/zulip/zulip/assets/87673068/319c3692-e963-41ca-a13b-8a3018b3a9df)

Here is what it looks like in ``Update Personal Message Flags``
![image](https://github.com/zulip/zulip/assets/87673068/2528c088-07db-462c-89e7-5fc5fbe20d28)